setup.py

```diff
@@ -13,6 +14,27 @@
     ]
 )
 
+install_requires = [
+    "setuptools",
+    # -*- Extra requirements: -*-
+    "collective.easyform",
+]
+test_requires = [
+    "plone.app.testing",
+    # Plone KGS does not use this version, because it would break
+    # Remove if your package shall be part of coredev.
+    # plone_coredev tests as of 2016-04-01.
+    "plone.testing>=5.0.0",
+    "plone.app.contenttypes",
+    "plone.app.robotframework[debug]",
+]
+# We need old versions of some dependencies to support Python 2
+if sys.version_info < (3,):
+    install_requires += ["Authlib<1", "simple-salesforce<1"]
+    test_requires += ["vcrpy<4"]
+else:
+    install_requires += ["simple-salesforce"]
+    test_requires += ["vcrpy"]
```

Comment:
We could express the same requirements with [environment markers](https://peps.python.org/pep-0508/#environment-markers):
```
    'Authlib<1; python_version<"3"',
    'simple-salesforce<1; python_version<"3"',
    'simple-salesforce; python_version>"3"',
```
for `install_requires` and
```
    'vcrpy<4; python_version<"3"',
    'vcrpy; python_version>"3"',
```
for `test_requires`.
The advantage is that information regarding dependency versions and python version is not lost when a wheel is built.
It will require `setuptools` v36.2 (released Jul 13th 2017) or later.

Reply:
Thanks, I knew about environment markers but forgot it's possible to list the same package twice with different markers. I'll try to make this change.